apps/server/src/config/server.ts

```diff
@@ -0,0 +1,27 @@
+import 'dotenv/config';
+
+import { AppConfig } from 'common';
+
+export const getServerConfig = () => {
+	return {
+		port: process.env.PORT || 3030,
+		debug_mode: process.env.DEBUG || false,
```

Comment:
for boolean env var, this should look like `var.toLowerCase() === 'true'`, so the comparison returns a boolean, and doesn't just assume all strings are turthy e.g. 'false'

Reply:
thanks for catching this. actually removed this debug_mode as there's already a `LOG_LEVEL` we can set as debug when needed